lms/static/js/discussions_management/views/divided_discussions.js

```diff
@@ -14,53 +15,54 @@
                 },
 
                 /**
-                * Returns the cohorted discussions list.
+                * Returns the divided discussions list.
                 * @param {string} selector - To select the discussion elements whose ids to return.
-                * @returns {Array} - Cohorted discussions.
+                * @returns {Array} - Divided discussions.
                 */
-                getCohortedDiscussions: function(selector) {
+                getDividedDiscussions: function(selector) {
                     var self = this,
-                        cohortedDiscussions = [];
+                        dividedDiscussions = [];
 
                     _.each(self.$(selector), function(topic) {
-                        cohortedDiscussions.push($(topic).data('id'));
+                        dividedDiscussions.push($(topic).data('id'));
                     });
-                    return cohortedDiscussions;
+                    return dividedDiscussions;
                 },
 
                 /**
-                * Save the cohortSettings' changed attributes to the server via PATCH method.
+                * Save the discussionSettings' changed attributes to the server via PATCH method.
                 * It shows the error message(s) if any.
                 * @param {object} $element - Messages would be shown before this element.
                 * @param {object} fieldData - Data to update on the server.
                 */
                 saveForm: function($element, fieldData) {
                     var self = this,
-                        cohortSettingsModel = this.cohortSettings,
+                        discussionSettingsModel = this.discussionSettings,
                         saveOperation = $.Deferred(),
                         showErrorMessage;
-
-                    showErrorMessage = function(message, $element) {
+                    showErrorMessage = function(message) {
```

Comment:
It is using the element from the parent function.  The one that is being input at line 38.